packages/@sanity/cli/src/commands/functions/logsFunctionsCommand.ts

```diff
@@ -23,6 +24,9 @@ Examples
 
   # Delete all logs for Sanity Function
   sanity functions logs --name echo --delete
+
+  # Watch for new logs (streaming mode)
+  sanity functions logs --name echo --watch
```

Comment:
is `--name` supposed to be here? sometimes I get confused with the conventions of `sanity-run`.

if it is valid, should we list it under options above?

Reply:
Good catch. The help is wrong.